ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-redesigned.component.html

```diff
@@ -7,7 +7,7 @@
 >
     <ng-container *ngIf="isLoaded && model">
         <section class="section">
-            <ddp-subject-panel></ddp-subject-panel>
+            <ddp-subject-panel [invitationId]="invitationId"></ddp-subject-panel>
```

Comment:
Theoretically, we can make invitationId as optional input and in case it wasn't passed try to find it in the session, we can do it, but still, I'd prefer to have it as an input for the selected user via a new prism in angio since we don't put invitationId into session in angio, only in testboston, wdyt?

Reply:
I updated subjectPanel component, now it takes just one argument - subject instead of lists of inputs. This helps to not modify other studies